keps/sig-auth/266-kubelet-client-certificate-bootstrap-rotation/README.md

```diff
@@ -0,0 +1,156 @@
+# KEP-266: Kubelet client certificate bootstrap and rotation
+
+<!-- toc -->
+- [Release Signoff Checklist](#release-signoff-checklist)
+- [Summary](#summary)
+  - [Goals](#goals)
+- [Proposal](#proposal)
+  - [During Kubelet Boot Sequence](#during-kubelet-boot-sequence)
+  - [As Expiration Approaches](#as-expiration-approaches)
+  - [Certificate Approval](#certificate-approval)
+  - [Test Plan](#test-plan)
+  - [Graduation Criteria](#graduation-criteria)
+  - [Upgrade / Downgrade Strategy](#upgrade--downgrade-strategy)
+  - [Version Skew Strategy](#version-skew-strategy)
+- [Implementation History](#implementation-history)
+<!-- /toc -->
+
+## Release Signoff Checklist
+
+- [x] Enhancement issue in release milestone, which links to KEP dir in [kubernetes/enhancements] (not the initial KEP PR)
+- [x] KEP approvers have approved the KEP status as `implementable`
+- [x] Design details are appropriately documented
+- [x] Test plan is in place, giving consideration to SIG Architecture and SIG Testing input
+- [x] Graduation criteria is in place
+- [x] "Implementation History" section is up-to-date for milestone
+- [x] Supporting documentation e.g., additional design documents, links to mailing list discussions/SIG meetings, relevant PRs/issues, release notes
+- [x] User-facing documentation has been created at https://kubernetes.io/docs/tasks/tls/certificate-rotation/
+
+## Summary
+
+Currently, a kubelet has a certificate/key pair that authenticates the kubelet to the kube-apiserver.
```

Comment:
Do we want to cross link this to server side certificate rotation KEP?

Reply:
I will once I port it to a KEP